test/xpu/run_test_with_skip.py

```diff
@@ -941,6 +941,12 @@ def launch_test(test_case, skip_list=None, exe_list=None):
     "test_conj_view_to_sparse_xpu_complex64",
     "test_neg_conj_view_to_sparse_xpu_complex128",
     "test_neg_view_to_sparse_xpu_float64",
+
+    # AssertionError: False is not true : Argument 0 during forward call unexpectedly materializes. 
+    # Either set `supports_cow_input_no_materialize_forward=False` in this operation's OpInfo, 
+    # add the arg to the OpInfo's `allow_cow_input_materialize_forward` list, 
+    # or change the implementation to avoid materialization.
+    "test_cow_input_cdist_xpu_float32",
```

Comment:
It should be a case of `test_ops::TestCompositeCompliance`. But the suite is not enabled here, but enabled in fine gran cases? Why add skip here?

Reply:
I see, you add the suite here.